packages/-ember-data/tests/unit/store/adapter-interop-test.js

```diff
@@ -32,9 +32,12 @@ module('unit/store/adapter-interop - DS.Store working with a DS.Adapter', functi
   });
 
   test('Adapter can be set as a name', function(assert) {
-    store = createStore({ adapter: 'application' });
+    const env = setupStore({ adapter: 'custom-adapter' });
```

Comment:
I'm sure you already saw this in #6347, but this test was deleted in that PR.

Reply:
The removed test was the version of this that set it to an instance, not the one that set it to a string. It does refactor it to not use setupStore/createStore though :)